accela_rest_sdk/accela_rest/rest_client.py

```diff
@@ -17,6 +17,14 @@ def get(self, path, params, auth_type):
 
         return response
 
+    def post(self, path, data, params, auth_type):
+        """ Post request. """
+        url = self.config['API_ENDPOINT'] + path
```

Comment:
should use urljoin in case the path or env var have unexpected slashes.

Reply:
👍  6707aed